core/src/banking_stage.rs

```diff
@@ -1915,6 +1972,31 @@ impl BankingStage {
             "packet_conversion",
         );
 
+        // Filter out extraneous consensus transactions
```

Comment:
it would be nice to include this `_filter_extraneous_votes_time` in the `slot_metrics_tracker`